desktop/src/main.rs

```diff
@@ -102,6 +104,14 @@ struct Opt {
         arg_enum
     )]
     power: PowerPreference,
+
+    /// (Optional) HTTP Proxy to use when loading movies via URL
+    #[clap(long, case_insensitive = true)]
+    http_proxy: Option<std::string::String>,
+
+    /// (Optional) HTTPS Proxy to use when loading movies via URL
+    #[clap(long, case_insensitive = true)]
+    https_proxy: Option<std::string::String>,
```

Comment:
These could be `Option<Url>` and clap will automatically try to parse them using `Url::parse`.